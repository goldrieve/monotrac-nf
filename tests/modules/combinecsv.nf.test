nextflow_process {

    name "Test Process COMBINE_CSV"
    script "main.nf"
    process "COMBINE_CSV"

    test("Should combine AA count csvs") {

        when {
            process {
                """
                input[0] = file("${projectDir}/tests/data/SVP_test_AA_counts.csv")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }
    }
}