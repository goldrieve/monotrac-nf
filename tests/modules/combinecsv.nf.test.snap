{
    "Should combine AA count csvs": {
        "content": [
            {
                "0": [
                    "combined.csv:md5,cb8a28fe497e791765fffa7b577a7430"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.7"
        },
        "timestamp": "2025-10-06T14:43:12.010990942"
    }
}